apps/web/pages/v2/apps/installed/index.tsx

```diff
@@ -0,0 +1,9 @@
+function RedirectPage() {
```

Comment:
We want to make sure that `/apps/installed` goes through and get the user to `/apps/installed/calendar` as the first item in the side menu.